txpipe/twopoint_fourier.py

```diff
@@ -439,11 +446,16 @@ def choose_ell_bins(self, pixel_scheme, f_sky):
         # user can specify the bandwidth, or we can just use
         # the maximum sensible value of Delta ell.
         nlb = nlb if nlb>0 else max(1,int(2 * np.pi / width))
-
+        '''
+        
         # The subclass of NmtBin that we use here just adds some
         # helper methods compared to the default NaMaster one.
         # Can feed these back upstream if useful.
-        ell_bins = MyNmtBin(int(pixel_scheme.nside), nlb=nlb)
+
+        # Creating the ell binning from the edges using this Namaster constructor.
+        edges = np.geomspace(self.config['ell_min'], self.config['ell_max'], self.config['n_ell'])
+        edges_int = edges.astype(int)
```

Comment:
The problem with this is that sometimes, after casting to `int`, the result of `geomspace` will have repeated values. What I usually do is:
```
edges = np.unique(np.geomspace(lmin, lmax, n_ell).astype(int))
```

Reply:
Great thanks a lot David for reviewing this. I added your solution to the repeating values, and will now look at the conflicts with master.